.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@1.0.3
+  flutter: circleci/flutter@1.1.0
+
+jobs:
+  build-dev:
+    executor:
+      name: android/android-machine
+      resource-class: medium
+
+    steps:
+      - checkout
+
+      - flutter/install_sdk: # Install Flutter SDK
+          version: 3.7.9
+
+      - flutter/install_pub:
+          cache-version: v1
+
+      - run:
+          name: Initializing the Config variables
+          command: |
+            echo "$CONFIG" | base64 --decode > lib/config.dart
+            echo "$FIREBASE_OPTIONS" | base64 --decode > lib/firebase_options.dart
+
+      - run:
+          name: Update App Name
+          command: |
+            sed -i "s/android:label=\".*\"/android:label=\"$APP_NAME\"/" android/app/src/main/AndroidManifest.xml
+
+      - run:
+          name: Initializing the Local properties variables
+          command: |
+            TAG_NAME=$(echo $CIRCLE_TAG | cut -d'-' -f1)
+            VERSION_CODE=$(echo $TAG_NAME | cut -d'v' -f2)
+            echo "flutter.versionName=$TAG_NAME 
+            flutter.versionCode=$VERSION_CODE
+            flutter.buildMode=release
+            flutter.sdk=/home/circleci/development/flutter
+            " > android/local.properties
+
+      - run:
+          name: Initializing the KEY properties variables
+          command: |
+            echo "$KEY_PROPERTIES" | base64 --decode > android/key.properties
+
+      - run:
+          name: Initializing the key store variables
+          command: |
+            echo "$KEYSTORE" | base64 --decode > android/app/keystore.jks
+
+      - run:
+          name: Add Google services to App
+          command: |
+            echo "$GOOGLE_APPLICATION_CREDENTIALS" | base64 --decode > android/app/google-services.json
+
+      - run:
+          name: Assemble release build
+          command: |
+            TAG_NAME=$(echo $CIRCLE_TAG | cut -d'-' -f1)
+            VERSION_CODE=$(echo $TAG_NAME | cut -d'v' -f2)
+            flutter build apk --split-per-abi --release --build-name=$VERSION_CODE
```

Comment:
@creativeprashant007  let's not split the apks, as it might not run on everyone's device